VTEX - Checkout API.json

```diff
@@ -16381,6 +17101,17 @@
                     "type": "string",
                     "default": "application/json"
                 }
+            },
+            "individualShippingEstimates": {
+                "name": "individualShippingEstimates",
+                "in": "query",
+                "description": "Shows the product estimate shipping date in the `shippingEstimateDate` field from the `orderForm`. For instance, three business days will be represented `3bd`.",

```

Comment:
> For instance, three business days will be represented `3bd`.

This sentence explains how `shippingEstimateDate` is represented in the orderForm, right? The actual field you're describing is a boolean, so the first time I read this I thought it was confusing. I would suggest clarifying this sentence or even removing it from here. It might be worth linking a specific doc about the orderForm structure where they can get more details about how `shippingEstimateDate` appears there.

Reply:
This comment is for every line where this parameter appears